starter-workflows/code-scanning/frogbot-fix.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: "Frogbot Fix"
+on:
+  push:
+    branches:
+      # Creating fix pull requests will be triggered by any push to one of the these branches.
```

Comment:
Creating fix pull requests will be triggered by any push to one of the these branches.
-->
The scanning and creation of pull requests with fixes are triggered by pushing code to one of the these branches.

Reply:
done